eidas-saml-parser/src/dist/config.yml

```diff
@@ -16,17 +16,12 @@ logging:
   appenders:
     - type: ${LOGGING_APPENDER:-logstash-console}
 
-connectorMetadataConfiguration:
-  url: ${CONNECTOR_NODE_METADATA_URL:-https://DefaultNotUsed}
-  expectedEntityId: ${CONNECTOR_NODE_ENTITY_ID:-DefaultNotUsed}
-  jerseyClientName: connector-metadata-client
-  maxRefreshDelay: ${METADATA_REFRESH_DELAY:-600000}
-  trustStore:
-    type: ${TRUSTSTORE_TYPES:-encoded}
-    store: ${CONNECTOR_NODE_METADATA_TRUSTSTORE:-DefaultNotUsed}
-    password: ${CONNECTOR_NODE_METADATA_TRUSTSTORE_PASSWORD:-marshmallow}
+metatronUri: ${METATRON_URL:-https://DefaultNotUsed}
+
+url: http://foo.com
```

Comment:
I don't really know what this does ...

Reply:
Looks like it's not used anywhere. I think you can drop it, as well as the getter in `EidasSamlParserConfiguration`